### ISSUE TITLE
Feature Request: Bulk Selection for accepting tasks

### ISSUE THREAD

**Issue by aremai** [TheHive-Project/TheHive#533]
### Request Type
Feature Request

Would it be possible to implement some sort of bulk selection for accepting tasks?
As an analyst I'm supposed to report at specific times my SOC activities, we handle this through tasks. Closing tasks is done individually, but accepting (taking over) is done in the morning.
Would be cool if there was some bulk action, or at least that I could open the tasks in different tabs.....

Many thanks in advance and please keep up the terrific work that you're doing!

**Comment by aremai** [comment]
hello,

is there an update for this?
it would be really nice to have this added to the hive ;) would make my life a little easier!

thanks,

**Comment by gekkeharry13** [comment]
Plus one for this feature request!

**Comment by aremai** [comment]
Afraid to ask if there's actually an update for this? We still badly need and want this :(

**Comment by nadouani** [comment]
Hello, @aremai @jeffrey-e do you mean you need a bulk start, assign etc... tasks? Just to understand the idea. Thanks

**Comment by jeffrey-e** [comment]
yes, that would be the case for me indeed. Just like alerts 👍 and maybe assign all tasks of a certain case to the same person.

**Comment by aremai** [comment]
@nadouani yes, exactly. especially assigning and closing of tasks would be grand! Thanks :)
